blockchain/fullblocktests/generate.go

```diff
@@ -2133,3 +2183,134 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 
 	return tests, nil
 }
+
+func GenerateHeaders() (generator *testGenerator, tests [][]TestInstance) {
```

Comment:
Could you leave a comment here about how `GenerateHeaders` works? It's not intuitive because the function itself is not really generating headers, rather it's generating blocks that'll only be used for headers.

Reply:
Okay I will add that